hyntatmta/code/modules/clothing/karma/karma_clothing.dm

```diff
@@ -0,0 +1,212 @@
+//Here you can see the karma outfits, which you can buy from karma shop
+//DIO's outfit
+/obj/item/clothing/head/atmta/dio
+	name = "DIO's heart headband"
+	desc = "Why is there a heart on this headband? The World may never know."
```

Comment:
Okay, can someone else please appreciate these Jojo references? I don't even watch it but these are gold